initials.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
```

Comment:
Great job! The only thing I would add is a comment with your name and the function of this program since it's good practice to do so. Overall, great job though! 👍